strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/users/UserControllerTestIT.java

```diff
@@ -268,7 +267,7 @@ void updateUser(String acceptHeader,
                .asString();
 
         // retrieve newly created user and store the objectId
-        UserData createdUser = retrieveUserByName(test.getUsername());
+        User createdUser = retrieveUserByName(test.getUsername());
         assertEquals(username, createdUser.getUsername());
 
         logger.info("Users before update: ->>>>>> ");
```

Comment:
Perhaps we no longer need this...? Most certainly, not as an `info`...

Reply:
fixed